sql.go

```diff
@@ -38,3 +39,10 @@ func (uuid *UUID) Scan(src interface{}) error {
 
 	return nil
 }
+
+// Value implements sql.Valuer so that UUIDs can be written to databases
+// transparently. Currently, UUIDs map map to strings. Please consult
```

Comment:
I think you have one too many maps in there.

Sorry for my long delay, I am not getting notifications from github for some reason